site/index.html

```diff
@@ -12,23 +12,7 @@
             <a href="docs.html">Documentation</a>
         </div>
         <div class="content">
-            <p>An engine for driving <a href="https://en.wikipedia.org/wiki/Category:Match_3_games">a "match three" game</a>, written in Javascript. Allows for plenty of customization, but defaults to mimic the match three system found in <a style="font-style: italic;" href="https://www.youtube.com/watch?v=GGWdqf1BnXo&t=13s">Gems of War</a>. This app is meant to be used as a core engine for a match three game, meaning that it contains no business logic concerning turns, or stats. This module only handles the moving, matching, removing, replenishing, and shuffling of "orbs". Orbs are the units that populate the board, and must be matched in groups of three or more to be removed from the board.</p>
-            <h2>Features</h2x>
-            <p>This list highlights a typical turn in a game of a match three style game. It also conveniently highlights the features of this module.</p>
-            <ol>
-                <li>Populates an initial board of orbs in a state that has no matches currently, but has at least one potential match.</li>
-                <li>Orb values are customizable. They default to integers 0 through 7.</li>
-                <li>Board sizes are customizable. They default to 8x8 in size.</li>
-                <li>Boards support "swapping", which move one orb to another adjacent square (not diagonal) to attempt a match.</li>
-                <li>Boards scan for matches after swap calls. Any locations on the board that have a consecutive 3+ run of the same orb in a row are "matched".</li>
-                <li>Matched orbs are tallied, and removed. You are given an object informing you of what orbs were removed, and where, for animation purposes.</li>
-                <li>Matched orbs are replaced from above with new orbs (gravity defaults to "down"). What type of orbs drop in is random by default, but can be configured.</li>
-                <li>After new orbs drop in, more matches are searched for. Any matches found are returned to you for animation purposes.</li>
-                <li>This cycle continues until there are no more matches available.</li>
-                <li>If there are zero potential matches, the board is shuffled until there are no current board matches, but at least one potential match.</li>
-                <li>Control of the board is returned to the consumer.</li>
-            </ol>
-            
+            <object data="README.html" type="text/html"></object>
```

Comment:
I found it going through stack overflow, but I can't find the link. The problem I have now is that there is a scroll bar i can't get rid of. I think `<object>` is similar to/an old version of `<iframe>`.